machines/store.ts

```diff
@@ -33,6 +42,8 @@ const model = createModel(
         requester,
       }),
       STORE_ERROR: (error: Error, requester?: string) => ({ error, requester }),
+      TAMPERED_VC: (error: Error, requester?: string) => ({ error, requester }),
```

Comment:
Can we relook at the TAMPERED_VC event signature once, as both the requester & error is not getting used in tthe event?
This also can follow like 
TAMPERED_VC: () => ({})

if the params passed are not getting used

Reply:
Refactored this.